README.md

```diff
@@ -319,6 +319,41 @@ td.fetchUserSegments(token, successCallback, errorCallback)
 ```
 *N.B.* This feature is not enabled on accounts by defaut, please contact support for more information.
 
+### Treasure#fetchUserSegments(options, success, failure)
+
+**Parameters:**
+
+* **options** : Object (required) - User Segment object 
+  * **options.audienceToken** : String or Array (required) - Audience Token(s) for the userId
+  * **options.keys** : Object (optional) - Key Value to be sent for this segment
+* **success** : Function (optional) - Callback for receiving the user key and segments
```

Comment:
Do we want to note the expected params here?

Reply:
I don't think we note them in any other example block (they all follow this pattern).